30-remote-site-setup.sh

```diff
@@ -2,7 +2,13 @@
 
 BOSCO_KEY=/etc/osg/bosco.key
 # $REMOTE_HOST needs to be specified in the environment
-REMOTE_HOST_KEY=`ssh-keyscan -H "$REMOTE_HOST"`
+remote_fqdn=${REMOTE_HOST%:*}
+if [[ "$REMOTE_HOST" =~ .*:[0-9]*$ ]]; then
```

Comment:
```suggestion
if [[ $REMOTE_HOST =~ :[0-9]+$ ]]; then
```

- minor: no `""`s needed on left-hand side in `[[ ... ]]`
- a regex matches anywhere in the string, so a leading `.*` doesn't do anything here
- `*` means the port can be an empty string; use a `+` requires it be non-empty